modules/nixos/buildbot-master/default.nix

```diff
@@ -0,0 +1,47 @@
+{ config, inputs, ... }:
+let
+  buildbotSecrets.sopsFile = ./secrets.yaml;
+in
+{
+  imports = [
+    inputs.buildbot-nix.nixosModules.buildbot-master
+  ];
+
+  services.nginx.virtualHosts."buildbot.nix-community.org" = {
+    enableACME = true;
+    forceSSL = true;
+  };
+
+  services.telegraf.extraConfig.inputs.prometheus.urls = [
+    "http://localhost:8011/metrics"
+  ];
+
+  sops.secrets.github-oauth-secret = buildbotSecrets;
+  sops.secrets.github-token = buildbotSecrets;
+  sops.secrets.github-webhook-secret = buildbotSecrets;
+  sops.secrets.nix-workers = buildbotSecrets;
+
+  services.buildbot-nix.master = {
+    enable = true;
+    buildSystems = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
+    domain = "buildbot.nix-community.org";
+    workersFile = config.sops.secrets.nix-workers.path;
+    github = {
+      tokenFile = config.sops.secrets.github-token.path;
```

Comment:
@Mic92 What permissions are needed for this token and the `nix-community-buildbot` user?

Reply:
I guess we need to make `nix-community-buildbot` an org owner otherwise we'd need manually add it as a collaborator to each repo.

Reply:
Yeah. looks like it.